data/ecosystems/c/chia.toml

```diff
@@ -2052,3 +2052,291 @@ url = "https://github.com/zodi1337/Raspberry-HDD-Mining-Projects"
+
+[[repo]]
+ url ="https://github.com/wzy1985/chia-plot-mover"
+
+[[repo]]
+ url ="https://github.com/SlowestTimelord/nftsonchia"
+
+[[repo]]
+ url ="https://github.com/tjallingk/ChiaSyncBench"
+
+[[repo]]
+ url ="https://github.com/mpwsh/chia-models-rs"
+
+[[repo]]
+ url ="https://github.com/konstantin-v03/trade_bot"
+
+[[repo]]
+ url ="https://github.com/AppleOfEnlightenment/core"
+
+[[repo]]
+ url ="https://github.com/AppleOfEnlightenment/wallets"
+
+[[repo]]
+ url ="https://github.com/Open-Market-Dev-Club/spriggan-marketplace-publishing-dapp"
+
+[[repo]]
+ url ="https://github.com/lmtlss/chia_reference_wallet_ios"
+
+[[repo]]
+ url ="https://github.com/ageorge95/spare-blockchain"
+
+[[repo]]
+ url ="https://github.com/lmtlss/clvm_tools_rs"
+
+[[repo]]
+ url ="https://github.com/SutuLabs/ChiaNameServiceRegistrar"
+
+[[repo]]
+ url ="https://github.com/ageorge95/lotus-blockchain_BAREBONE_v1.5.0"
+
+[[repo]]
+ url ="https://github.com/steppsr/discord-faucet-bot"
+
+[[repo]]
+ url ="https://github.com/asicminer8/plotctl"
+
+[[repo]]
+ url ="https://github.com/inan0812/goji-blockchain"
+
+[[repo]]
+ url ="https://github.com/Fluffy9/Instaplot"
+
+[[repo]]
+ url ="https://github.com/AppleOfEnlightenment/icons"
+
+[[repo]]
+ url ="https://github.com/graemes/chiamon"
+
+[[repo]]
+ url ="https://github.com/Open-Market-Dev-Club/spriggan-shared"
+
+[[repo]]
+ url ="https://github.com/Yakuhito/tibet-analytics-api"
+
+[[repo]]
+ url ="https://github.com/ageorge95/flora-blockchain"
+
+[[repo]]
+ url ="https://github.com/nongyehezuoshe/bech32m-chia"
+
+[[repo]]
+ url ="https://github.com/keyspaceapp/swift-bip158"
+
+[[repo]]
+ url ="https://github.com/xorde-labs/docker-chia-node"
+
+[[repo]]
+ url ="https://github.com/Yakuhito/tibet-ui"
+
+[[repo]]
+ url ="https://github.com/jackygu2006/chialisp-doc"
+
+[[repo]]
+ url ="https://github.com/Open-Market-Dev-Club/spriggan-client-gui"
+
+[[repo]]
+ url ="https://github.com/SutuLabs/pawket-js-sdk"
+
+[[repo]]
+ url ="https://github.com/keyspaceapp/swift-pos"
+
+[[repo]]
+ url ="https://github.com/Open-Market-Dev-Club/chia-walletconnect"
+
+[[repo]]
+ url ="https://github.com/graemes/mownplow"
+
+[[repo]]
+ url ="https://github.com/hashgreen/cypher-chialisp"
+
+[[repo]]
+ url ="https://github.com/0xgorro/chitter"
+
+[[repo]]
+ url ="https://github.com/Open-Market-Dev-Club/spriggan-marketplace-backend"
+
+[[repo]]
+ url ="https://github.com/mikehw/chia-signing-tools"
+
+[[repo]]
+ url ="https://github.com/wallentx/chia-extension"
+
+[[repo]]
+ url ="https://github.com/foxypool/foxy-farmer"
+
+[[repo]]
+ url ="https://github.com/geoffwalmsley/agg_sig_me"
+
+[[repo]]
+ url ="https://github.com/davewolf678/snapshot"
+
+[[repo]]
+ url ="https://github.com/mx-pool/super-farmer"
+
+[[repo]]
+ url ="https://github.com/0xgorro/chia-token-gated-access"
+
+[[repo]]
+ url ="https://github.com/Yakuhito/tibet-analytics"
+
+[[repo]]
+ url ="https://github.com/iced-caffeine/py_dexie_tools"
```

Comment:
Can remove

Reply:
This is a tool to aggregate all the chia offer files on the dexie service.  It has everything to do with the Chia Network.

Reply:
in general we tend to only add stuff that's built on top of ecosystems. Let's just keep it given that the repo has the chia tag on it